cmd/consul-dataplane/main.go

```diff
@@ -245,3 +248,19 @@ func main() {
 		log.Fatal(err)
 	}
 }
+
+// readServiceIDFromFile reads the service ID from the file specified by the
+// -proxy-service-id-path flag.
+//
+// We do this here, rather than in the consuldp package's config handling,
+// because this option only really makes sense as a CLI flag (and we handle
+// all flag parsing here).
```

Comment:
I don't _think_ it'd make sense to add this to the config file, what do you think?